src/docs/perf/rendering/best-practices.md

```diff
@@ -24,7 +41,7 @@ when designing your app:
 * Avoid overly large single Widgets with a large `build()` function.
   Split them into different Widgets based on encapsulation but also on how
   they change:
-  * When `setState()` is called on a State, all descendent widgets will
```

Comment:
I believe that in this case "will" is correct.

Reply:
I don't think it's adding any clarity here. The sentence is still correct without it.